tests/test_edgedockerclient_int.py

```diff
@@ -59,13 +60,24 @@ def _create_container(self, client):
             mounts=[docker.types.Mount(volume_path, self.VOLUME_NAME)]
         )
         network_config = client.create_config_for_network(self.NETWORK_NAME)
-        client.create_container(
-            image_name,
-            name=self.CONTAINER_NAME,
-            networking_config=network_config,
-            host_config=host_config,
-            volumes=[volume_path],
-            environment=env_dict)
+        vm_os_type = platform.system().lower()
+        if (vm_os_type == 'darwin') or (vm_os_type == 'linux'):
+            client.create_container(
+                image_name,
+                name=self.CONTAINER_NAME,
+                networking_config=network_config,
+                host_config=host_config,
+                volumes=[volume_path],
+                environment=env_dict,
+                command='sleep 20s')
```

Comment:
why it need sleep on mac/ linux?

Reply:
We can sleep to keep the container running. And I think this could also apply to windows os_type instead using if/else with different logic